stdlib/public/SDK/Foundation/CharacterSet.swift

```diff
@@ -412,7 +412,17 @@ public struct CharacterSet : ReferenceConvertible, Equatable, Hashable, SetAlgeb
             $0.formIntersection(with: other)
         }
     }
-    
+
+    /// Returns the set of elements in the `CharacterSet` but not in another `CharacterSet`.
```

Comment:
This documentation line doesn't really parse well for me.

How about this?

```/// Returns a `CharacterSet` created by removing the contents of `other` from `self`. ```

and something similar for the mutating version.